cargo-insta/tests/main.rs

```diff
@@ -918,12 +923,13 @@ Hello, world!
     assert_snapshot!(test_insta_1_40_0.diff("src/snapshots/test_force_update_1_40_0__force_update.snap"), @r#"
     --- Original: src/snapshots/test_force_update_1_40_0__force_update.snap
     +++ Updated: src/snapshots/test_force_update_1_40_0__force_update.snap
-    @@ -1,8 +1,5 @@
+    @@ -1,8 +1,6 @@
     -
      ---
      source: src/lib.rs
     -expression: 
     +expression: "\"Hello, world!\""
+    +snapshot_type: text
```

Comment:
One tiny change before we merge is to rename this to `_kind` to be more idiomatic, given it'll become part of our interface